google/cloud/aiplatform/experimental/vertex_model/base.py

```diff
@@ -63,37 +64,80 @@ def f(*args, **kwargs):
 
         training_source = source_utils._make_class_source(obj)
 
-        source = source_utils._make_source(
-            cls_source=training_source,
-            cls_name=cls_name,
-            instance_method=method.__name__,
-        )
-
         with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
It seems like this context manager and the `script_path` creation can be moved below, right above where we write the source.